source/iterators.tex

```diff
@@ -476,11 +476,9 @@
 The only valid use of an
 \tcode{operator*}
 is on the left side of the assignment statement.
-\textit{Assignment through the same value of the iterator happens only once.}
+Assignment through the same value of the iterator happens only once.
```

Comment:
If we can't use blinking text here instead then I agree with removing the emphasis.

Reply:
```html
<marquee>Assignment through the same value of the iterator happens only once.</marquee>
```